clowdapp.yaml

```diff
@@ -292,7 +292,7 @@ parameters:
   value: "1"
 - description: Replica count for inventory processor pod
   name: INVENTORY_PROCESSOR_REPLICA_COUNT
-  value: "1"
+  value: "2"
```

Comment:
This will get affect in all the environments. Please close this PR and open one against app-interface. 
Add `INVENTORY_PROCESSOR_REPLICA_COUNT` parameter below [line 87](https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/data/services/insights/ros/deploy-clowder.yml#L87) and set the value to 2, it will help in increasing the inventory processor pod only in stage env.